notifyosd.zsh

```diff
@@ -45,7 +48,7 @@ function notifyosd-precmd() {
             ((cmd_secs=$cmd_end - $cmd_start))
         fi
 
-        if [ ! -z "$cmd" -a $cmd_secs -gt 10 ] && is_window_unfocused; then
+        if [ ! -z "$cmd" -a $cmd_secs -gt ${LONG_RUNNING_COMMAND_TIMEOUT:-10} ] && is_window_unfocused; then
```

Comment:
The substitution is required since this file is sourced once and the variable could be exported later on manually.